src/test/setup.h

```diff
@@ -1,19 +1,95 @@
 #if defined(WIN32) && defined(SNMALLOC_CI_BUILD)
 #  include <ds/bits.h>
+#  include <iostream>
+#  include <pal/pal.h>
 #  include <signal.h>
 #  include <stdlib.h>
+// Has to come after the PAL.
+#  include <DbgHelp.h>
+#  pragma comment(lib, "dbghelp.lib")
+
+void print_stack_trace()

```

Comment:
Do we want to export this in the PAL as the `PAL::error()` thing, and then just call that from the tests?  That would let us produce a stack trace from any PAL that supports it, from any of the checked error conditions.

Reply:
That makes sense.  We would need to stop calling abort().  Also, assert failures call abort.  So perhaps we need to override assert as well.  I'll create an issue for this, as I think it needs more discussion and design than I have time for at the moment.

Reply:
#89